### ISSUE TITLE
auto-sklearn with harder time-constraints

### ISSUE THREAD

**Issue by LeanderK** [automl/auto-sklearn#707]
Does auto-sklearn work with harder time-constraints assuming I am accepting a trade of accuracy?

What about constraining it to get results in minutes/seconds? I have quite a few cores available that I can commit to it, does auto-sklearn benefit from higher amount of n_jobs (30/50?).

The reasons is that need some baselines for a lot of dataset-versions so I can't commit a lot of time per individual dataset.

I currently have:

```
automl = autosklearn.classification.AutoSklearnClassifier(time_left_for_this_task=30, per_run_time_limit=5, n_jobs=n_cores)
```

I am getting: BudgetExhaustedException
